### ISSUE TITLE
While on PD charger, app show profile change from time to time

### ISSUE THREAD

**Issue by lucidnx** [seerge/g-helper#40]
Hello!
On latest version (0.14.0.0) I have noticed that when battery is full, I am receiving OSD notification that profile has been changed to "Balanced" from time to time. I am not sure why.
Maybe when the battery reaches 100% something happens on the laptop and GHelper thinks the laptop's power source has changed?

**Comment by seerge** [comment]
Yes, i have also noticed that. It seem to trigger every time charging stops /starts.
I have uploaded a fixed version over old 0.14 https://github.com/seerge/g-helper/releases/tag/v0.14
so it should be fine now

**Comment by lucidnx** [comment]
seems to be okay now, thanks :)

**Comment by seerge** [comment]
Great
